modules/standard/items/items_controller.py

```diff
@@ -136,13 +136,13 @@ def sort_items(self, search, items):
 
     def get_by_item_id(self, item_id, ql=None):
         if ql:
-            return self.db.query_single("SELECT * FROM aodb WHERE (highid = ? OR lowid = ?) AND (lowql <= ? AND highql >= ?) ORDER BY highid DESC LIMIT 1", [item_id, item_id, ql, ql])
+            return self.db.query_single("SELECT * FROM aodb WHERE (highid = ? OR lowid = ?) AND (lowql <= ? AND highql >= ?) ORDER BY highid = ? LIMIT 1", [item_id, item_id, ql, ql, item_id])
         else:
-            return self.db.query_single("SELECT * FROM aodb WHERE highid = ? OR lowid = ? ORDER BY highid DESC LIMIT 1", [item_id, item_id])
+            return self.db.query_single("SELECT * FROM aodb WHERE highid = ? OR lowid = ? ORDER BY highid = ? LIMIT 1", [item_id, item_id])
 
     def find_by_name(self, name, ql=None):
         if ql:
-            return self.db.query_single("SELECT * FROM aodb WHERE name = ? AND lowql <= ? AND highql >= ? ORDER BY highid DESC LIMIT 1", [name, ql, ql])
+            return self.db.query_single("SELECT * FROM aodb WHERE name = ? AND lowql <= ? AND highql >= ? ORDER BY highid DESC LIMIT 1", [name, ql, ql, item_id])
```

Comment:
Here you added the 4th param, but didnt change the order by to "ORDER BY highid = ?"
in my tests, i removed the 4th param.